<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="dcterms.date" content="2024-12-28">

<title>Publications – Ann Wells, Ph.D.&nbsp;| Open Science | Data Science | Lupus Researcher</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-dashboard/quarto-dashboard.js"></script>
<script src="site_libs/quarto-dashboard/stickythead.js"></script>
<script src="site_libs/quarto-dashboard/datatables.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/pdfmake.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/vfs_fonts.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/web-components.js" type="module"></script>
<script src="site_libs/quarto-dashboard/components.js"></script>
<link href="site_libs/quarto-dashboard/datatables.min.css" rel="stylesheet" kdttablesentinel="true">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-41BY0H900V"></script>

<script type="text/javascript">

window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', 'G-41BY0H900V', { 'anonymize_ip': true});
</script>


</head>

<body class="quarto-dashboard nav-fixed fullcontent">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/Ann.png" alt="" class="navbar-logo">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Welcome</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./publications.html" aria-current="page"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./open-science.html"> 
<span class="menu-text">Open Science</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://orcid.org/0000-0002-9720-0038"> <i class="bi bi-file-earmark-fill" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/annwells/"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/annwellsphd/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:ann.wells@jax.org"> <i class="bi bi-send" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./CV-Ann-2024.pdf"> 
<span class="menu-text">CV</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
<header id="quarto-dashboard-header">
<nav class="navbar navbar-expand-md slim" data-bs-theme="dark">
  <div class="navbar-container container-fluid">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#dashboard-collapse" aria-controls="dashboard-collapse" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>    


    <div class="navbar-brand-container">       

      <div class="navbar-title">
        <div class="navbar-title-text"><a href="#">Publications</a></div>
        
        
      </div>
    </div>
<div id="dashboard-collapse" class="navbar-collapse collapse"><ul class="navbar-nav navbar-nav-scroll me-auto" role="tablist"><li class="nav-item" role="presentation"><a id="tab-section" class="nav-link active" data-bs-toggle="tab" role="tab" data-bs-target="#section" data-scrolling="true" href="#section" aria-controls="section" aria-selected="true"><span class="nav-link-text">2024</span></a></li><li class="nav-item" role="presentation"><a id="tab-section-1" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#section-1" data-scrolling="true" href="#section-1" aria-controls="section-1" aria-selected="false"><span class="nav-link-text">2022</span></a></li><li class="nav-item" role="presentation"><a id="tab-section-2" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#section-2" data-scrolling="true" href="#section-2" aria-controls="section-2" aria-selected="false"><span class="nav-link-text">2021</span></a></li><li class="nav-item" role="presentation"><a id="tab-section-3" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#section-3" data-scrolling="true" href="#section-3" aria-controls="section-3" aria-selected="false"><span class="nav-link-text">2018</span></a></li><li class="nav-item" role="presentation"><a id="tab-section-4" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#section-4" data-scrolling="true" href="#section-4" aria-controls="section-4" aria-selected="false"><span class="nav-link-text">2011</span></a></li><li class="nav-item" role="presentation"><a id="tab-section-5" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#section-5" data-scrolling="true" href="#section-5" aria-controls="section-5" aria-selected="false"><span class="nav-link-text">2010</span></a></li></ul></div></div>
</nav>
</header></header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-layout-custom page-navbar quarto-dashboard-content bslib-gap-spacing html-fill-container dashboard-scrolling quarto-dashboard-pages">
<!-- sidebar -->
<!-- margin-sidebar -->
<!-- main -->

<div class="bslib-grid-item html-fill-item tab-content bslib-grid" data-layout="flow" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div id="section" class="dashboard-page tab-pane show active html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="Transcriptome Analysis Reveals Organ‑Specific Effects of 2‑Deoxyglucose Treatment in Healthy Mice" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Transcriptome Analysis Reveals Organ‑Specific Effects of 2‑Deoxyglucose Treatment in Healthy Mice</div></div>
<div class="card-body html-fill-item html-fill-container">
<p><strong>Ann E. Wells</strong>, John J. Wilson, John D. Sears, Jian Wei, Sarah E. Heuer, Raghav Pandey, Mauro W. Costa, Catherine C. Kaczorowski, Derry C. Roopenian, Chih‑Hao Chang, Gregory W. Carter. (2024) Transcriptome Analysis Reveals Organ‑Specific Effects of 2‑Deoxyglucose Treatment in Healthy Mice. PLOS ONE 19(3): e0299595. https://doi.org/10.1371/journal.pone.0299595. <a href="https://doi.org/10.1371/journal.pone.0299595" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p><strong>Objective:</strong> Glycolytic inhibition via 2-deoxy-D-glucose (2DG) has potential therapeutic benefits for a range of diseases, including cancer, epilepsy, systemic lupus erythematosus (SLE), and rheumatoid arthritis (RA), and COVID-19, but the systemic effects of 2DG on gene function across different tissues are unclear.</p>
<p><strong>Methods:</strong> This study analyzed the transcriptional profiles of nine tissues from C57BL/6J mice treated with 2DG to understand how it modulates pathways systemically. Principal component analysis (PCA), weighted gene co-network analysis (WGCNA), analysis of variance, and pathway analysis were all performed to identify modules altered by 2DG treatment.</p>
<p><strong>Results:</strong> PCA revealed that samples clustered predominantly by tissue, suggesting that 2DG affects each tissue uniquely. Unsupervised clustering and WGCNA revealed six distinct tissue-specific modules significantly affected by 2DG, each with unique key pathways and genes. 2DG predominantly affected mitochondrial metabolism in the heart, while in the small intestine, it affected immunological pathways.</p>
<p><strong>Conclusions:</strong> These findings suggest that 2DG has a systemic impact that varies across organs, potentially affecting multiple pathways and functions. The study provides insights into the potential therapeutic benefits of 2DG across different diseases and highlights the importance of understanding its systemic effects for future research and clinical applications.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="section-1" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section-1">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="Tissue Level Strain and Sex‑by‑Strain Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry Across Adipose, Skeletal Muscle, and Liver Tissue in Mice Fed a Standard Chow Diet" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Tissue Level Strain and Sex‑by‑Strain Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry Across Adipose, Skeletal Muscle, and Liver Tissue in Mice Fed a Standard Chow Diet</div></div>
<div class="card-body html-fill-item html-fill-container">
<p><strong>Ann E. Wells</strong>, William T. Barrington, Stephen Dearth, Nikhil Milind, Gregory W. Carter, David W. Threadgill, Shawn Campagna, Brynn Voy. Tissue Level Strain and Sex‑by‑Strain Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry Across Adipose, Skeletal Muscle, and Liver Tissue in Mice Fed a Standard Chow Diet. Metabolites. 2022 Apr 8;12(4):337. doi: 10.3390/metabo12040337. PMID: 35448524; PMCID: PMC9031494. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9031494/" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>Genetics play an important role in the development of metabolic diseases. However, the relative influence of genetic variation on metabolism is not well defined, particularly in tissues, where metabolic dysfunction that leads to disease occurs. We used inbred strains of laboratory mice to evaluate the impact of genetic variation on the metabolomes of tissues that play central roles in metabolic diseases. We chose a set of four common inbred strains that have different levels of susceptibility to obesity, insulin resistance, and other common metabolic disorders. At the ages used, and under standard husbandry conditions, these lines are not overtly diseased. Using global metabolomics profiling, we evaluated water-soluble metabolites in liver, skeletal muscle, and adipose from A/J, C57BL/6J, FVB/NJ, and NOD/ShiLtJ mice fed a standard mouse chow diet. We included both males and females to assess the relative influence of strain, sex, and strain-by-sex interactions on metabolomes. The mice were also phenotyped for systems level traits related to metabolism and energy expenditure. Strain explained more variation in the metabolite profile than did sex or its interaction with strain across each of the tissues, especially in liver. Purine and pyrimidine metabolism and pathways related to amino acid metabolism were identified as pathways that discriminated strains across all three tissues. Based on the results from ANOVA, sex and sex-by-strain interaction had modest influence on metabolomes relative to strain, suggesting that the tissue metabolome remains largely stable across sexes consuming the same diet. Our data indicate that genetic variation exerts a fundamental influence on tissue metabolism.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="section-2" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section-2">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr) minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="The Combined Analysis of Pleiotropy and Epistasis (CAPE)" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">The Combined Analysis of Pleiotropy and Epistasis (CAPE)</div></div>
<div class="card-body html-fill-item html-fill-container">
<p>Tyler AL, Emerson J, El Kassaby B, <strong>Wells AE</strong>, Philip VM, Carter GW. The Combined Analysis of Pleiotropy and Epistasis (CAPE). Methods Mol Biol. 2021;2212:55‑67. doi: 10.1007/978‑1‑0716‑0947‑7_5. PMID: 33733350. <a href="https://pubmed.ncbi.nlm.nih.gov/33733350/" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>Epistasis, or gene-gene interaction, contributes substantially to trait variation in organisms ranging from yeast to humans, and modeling epistasis directly is critical to understanding the genotype-phenotype map. However, inference of genetic interactions is challenging compared to inference of individual allele effects due to low statistical power. Furthermore, genetic interactions can appear inconsistent across different quantitative traits, presenting a challenge for the interpretation of detected interactions. Here we present a method called the Combined Analysis of Pleiotropy and Epistasis (CAPE) that combines information across multiple quantitative traits to infer directed epistatic interactions. By combining information across multiple traits, CAPE not only increases power to detect genetic interactions but also interprets these interactions across traits to identify a single interaction that is consistent across all observed data. This method generates informative, interpretable interaction networks that explain how variants interact with each other to influence groups of related traits. This method could potentially be used to link genetic variants to gene expression, physiological endophenotypes, and higher-level disease traits.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
<div class="card html-fill-item html-fill-container bslib-card" data-title="Effects of kinship correction on inflation of genetic interaction statistics in commonly used mouse populations" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Effects of kinship correction on inflation of genetic interaction statistics in commonly used mouse populations</div></div>
<div class="card-body html-fill-item html-fill-container">
<p>Tyler AL, El Kassaby B, Kolishovski G, Emerson J, <strong>Wells AE</strong>, Matthew Mahoney J, Carter GW. Effects of kinship correction on inflation of genetic interaction statistics in commonly used mouse populations. G3 (Bethesda). 2021 Jul 14;11(7):jkab131. doi: 10.1093/g3journal/jkab131. PMID: 33892506; PMCID: PMC8496251. <a href="https://pubmed.ncbi.nlm.nih.gov/33892506/" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>It is well understood that variation in relatedness among individuals, or kinship, can lead to false genetic associations. Multiple methods have been developed to adjust for kinship while maintaining power to detect true associations. However, relatively unstudied are the effects of kinship on genetic interaction test statistics. Here, we performed a survey of kinship effects on studies of six commonly used mouse populations. We measured inflation of main effect test statistics, genetic interaction test statistics, and interaction test statistics reparametrized by the Combined Analysis of Pleiotropy and Epistasis (CAPE). We also performed linear mixed model (LMM) kinship corrections using two types of kinship matrix: an overall kinship matrix calculated from the full set of genotyped markers, and a reduced kinship matrix, which left out markers on the chromosome(s) being tested. We found that test statistic inflation varied across populations and was driven largely by linkage disequilibrium. In contrast, there was no observable inflation in the genetic interaction test statistics. CAPE statistics were inflated at a level in between that of the main effects and the interaction effects. The overall kinship matrix overcorrected the inflation of main effect statistics relative to the reduced kinship matrix. The two types of kinship matrices had similar effects on the interaction statistics and CAPE statistics, although the overall kinship matrix trended toward a more severe correction. In conclusion, we recommend using an LMM kinship correction for both main effects and genetic interactions and further recommend that the kinship matrix be calculated from a reduced set of markers in which the chromosomes being tested are omitted from the calculation. This is particularly important in populations with substantial population structure, such as recombinant inbred lines in which genomic replicates are used.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="section-3" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section-3">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr) minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="Tissue Level Diet and Sex‑by‑Diet Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry on Adipose, Skeletal Muscle, and Liver tissue in C57BL6/J Mice" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Tissue Level Diet and Sex‑by‑Diet Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry on Adipose, Skeletal Muscle, and Liver tissue in C57BL6/J Mice</div></div>
<div class="card-body html-fill-item html-fill-container">
<p><strong>Ann E. Wells</strong>, William T. Barrington, Stephen Dearth, Amanda May, David W. Threadgill, Shawn Campagna, Brynn Voy. Tissue Level Diet and Sex‑by‑Diet Interactions Reveal Unique Metabolite and Clustering Profiles Using Untargeted Liquid Chromatography‑Mass Spectrometry on Adipose, Skeletal Muscle, and Liver tissue in C57BL6/J Mice. J Proteome Res. 2018 Mar 2;17(3):1077‑1090. doi: 10.1021/acs.jproteome.7b00750. Epub 2018 Feb 2. PMID: 29373032. <a href="https://pubmed.ncbi.nlm.nih.gov/29373032/" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>Dietary intervention is commonly used for weight loss or to improve health, as diet-induced obesity increases the risk of developing type 2 diabetes, hypertension, cardiovascular disease, stroke, osteoarthritis, and certain cancers. Various dietary patterns are associated with effects on health, yet little is known about the effects of diet at the tissue level. Using untargeted metabolomics, this study aimed to identify changes in water-soluble metabolites in C57BL/6J males and females fed one of five diets (Japanese, ketogenic, Mediterranean, American, and standard mouse chow) for 7 months. Metabolite abundance was examined in liver, skeletal muscle, and adipose tissue for sex, diet, and sex-by-diet interaction. Analysis of variance (ANOVA) suggests that liver tissue has the most metabolic plasticity under dietary changes compared with adipose and skeletal muscle. The ketogenic diet was distinguishable from other diets for both males and females according to partial least-squares discriminant analysis. Pathway analysis revealed that the majority of pathways affected play an important role in amino acid metabolism in liver tissue. Not surprisingly, amino acid profiles were affected by dietary patterns in skeletal muscle. Few metabolites were significantly altered in adipose tissue relative to skeletal muscle and liver tissue, indicating that it was largely stable, regardless of diet alterations. The results of this study revealed that the ketogenic diet had the largest effect on physiology, particularly for females. Furthermore, metabolomics analysis revealed that diet affects metabolites in a tissue-specific manner and that liver was most sensitive to dietary changes.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
<div class="card html-fill-item html-fill-container bslib-card" data-title="Optimizing Metabolic Health Through Precision Dietetics in Mice. Genetics" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Optimizing Metabolic Health Through Precision Dietetics in Mice. Genetics</div></div>
<div class="card-body html-fill-item html-fill-container">
<p>William T. Barrington, Phillip Wulfridge, <strong>Ann E. Wells</strong>, Carolina Mantilla Rojas, Selene Y.F. Howe, Amie Perry, Kunjie Hua, Michael Pellizzon, Kasper D. Hansen, Brynn Voy, Brian J. Bennett, Daniel Pomp, Andrew P. Feinberg, David W. Threadgill. (2017) Optimizing Metabolic Health Through Precision Dietetics in Mice. Genetics. 2018 Jan;208(1):399‑417. doi: 10.1534/genetics.117.300536. Epub 2017 Nov 20. PMID: 29158425; PMCID: PMC5753872. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5753872/" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>The incidence of diet-induced metabolic disease has soared over the last half-century, despite national efforts to improve health through universal dietary recommendations. Studies comparing dietary patterns of populations with health outcomes have historically provided the basis for healthy diet recommendations. However, evidence that population-level diet responses are reliable indicators of responses across individuals is lacking. This study investigated how genetic differences influence health responses to several popular diets in mice, which are similar to humans in genetic composition and the propensity to develop metabolic disease, but enable precise genetic and environmental control. We designed four human-comparable mouse diets that are representative of those eaten by historical human populations. Across four genetically distinct inbred mouse strains, we compared the American diet’s impact on metabolic health to three alternative diets (Mediterranean, Japanese, and Maasai/ketogenic). Furthermore, we investigated metabolomic and epigenetic alterations associated with diet response. Health effects of the diets were highly dependent on genetic background, demonstrating that individualized diet strategies improve health outcomes in mice. If similar genetic-dependent diet responses exist in humans, then a personalized, or “precision dietetics,” approach to dietary recommendations may yield better health outcomes than the traditional one-size-fits-all approach.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="section-4" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section-4">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="Modeling the effect of soluble fibrin on the immune‑tumor interaction" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Modeling the effect of soluble fibrin on the immune‑tumor interaction</div></div>
<div class="card-body html-fill-item html-fill-container">
<p><strong>A. E. Wells</strong>, S. A. Bewick, K. L. Kruse, R. C. Ward and J. P. Biggerstaff, “Modeling the effect of soluble fibrin on the immune‑tumor interaction.” Proceedings of the 2011 Biomedical Sciences and Engineering Conference: Image Informatics and Analytics in Biomedicine, Knoxville, TN, USA, 2011, pp.&nbsp;1‑4, doi: 10.1109/BSEC.2011.5872324. <a href="https://ieeexplore.ieee.org/document/5872324" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>Cancer is the second leading cause of death in the United States. It is believed that many people develop cancers in their lifetime but the immune system kills these cells without the need for outside treatments. In cancer progression, however, tumor cells may evade the immune system by a mechanism that is not fully understood. Soluble fibrin (sFn), a marker for disseminated intravascular coagulation, may play a role in evasion by tumor cells. As sFn becomes the predominant substance of the matrix surrounding the tumor cells it becomes immunosuppressive. This change from a predominantly collagen matrix to a sFn matrix could play a major role in tumor evasion. Previous research has shown that when sFn is present during the immune-tumor interaction monocyte receptors (MAC-1 and LFA-1) that, interact with CD54 on the tumor cell are altered. When sFn is bound to CD54 the LFA-1 receptor can no longer bind to the tumor cell, however, MAC-1 is still functional. If sFn binds to both, MAC-1 and CD54, then the monocyte is no longer functional against the tumor cell. A mathematical model was developed using a series of scenarios to create a set of continuous ordinary differential equations. These equations were coded in MATLAB to obtain simulations. Parameters were altered to show that the equations validated the experimental research previously published.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="section-5" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-section-5">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="Modeling the effect of tumor cell growth when in the presence of natural killer cells" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Modeling the effect of tumor cell growth when in the presence of natural killer cells</div></div>
<div class="card-body html-fill-item html-fill-container">
<p><strong>A. E. Wells</strong>, S. A. Bewick, K. L. Kruse, R. C. Ward and J. P. Biggerstaff, “Modeling the effect of tumor cell growth when in the presence of natural killer cells.” 2010 Biomedical Sciences and Engineering Conference, Oak Ridge, TN, USA, 2010, pp.&nbsp;1‑4, doi: 10.1109/BSEC.2010.5510820. <a href="https://ieeexplore.ieee.org/document/5510820" target="_blank">paper link</a></p>
<div class="h4">
Abstract
</div>
<p>Cancer is the second leading cause of death in the United States. It is believed that many humans develop cancers in their lifetime, but the immune system kills these cells without the need for outside treatments. In cancer progression, however, tumor cells may evade the immune system. The mechanism is not fully understood. Natural killer cells (NK) are one of many immune cells capable of tumor cell lysis. Specifically, the natural cytotoxicity receptors NKp46, NKp44 and NKp30, and the c-type lectin receptor NKG2D on NK cells, are crucial in lysing tumor cells. MHC Class-1 related chain A (MICA), one of several possible activating ligands for NKG2D, is broadly expressed on human tumor cells of epithelial origin. MICA, in combination with various matrix metalloproteinases (MMPs), appears to be involved in both tumor elimination and tumor growth. Tumor growth tends to increase in the presence of soluble MICA but decreases when high levels of MICA bind to a tumor cell. MMPs have been implicated in the shedding of MICA from the tumor surface. A mathematical model was developed to simulate the interactions between NK cells and tumor cells in the presence of MMPs. A set of twelve continuous partial differential equations were created based on data from published literature and solved. These simultaneous equations describe the relationship between twelve state variables and various parameters which will be determined through experimentation. Results of model simulations with three different initial concentration values of MMPs show how this biochemical affects the ability of NK cells to attack tumor cells.</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>


<div id="3ade8a4a-fb1d-4a6c-8409-ac45482d5fc9" class="hidden html-fill-item html-fill-container">

</div>
</div>
</div>
</div>


<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p><br> Copyright © 2024 Ann Wells, Ph.D.<br></p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>

<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (_event) {
  if (window.bslib.Card) {
    window.bslib.Card.initializeAllCards();
  }
}); 
</script>
  



</body></html>